### PR TITLE
feat(widget-recents): load requested number of spaces

### PR DESCRIPTION
**Pull request by adamweeks**
Removing the secondary space load due to the default limit and hard coded 25 being the same amount.

This also makes the first spaces request follow the limit passed in params.

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-217484

### CODE REVIEW

File: packages/node_modules/@webex/widget-recents/src/enhancers/setup.js [hunk 1]
```diff
@@ -148,11 +148,13 @@ function getInitialSpaces(props) {
        * and decrypt individually to give a good initial user experience.
        */
       props.fetchSpacesEncrypted(sparkInstance, {
-        conversationsLimit: 25
+        conversationsLimit: props.spaceLoadCount
```
Comment (taymoork2) [inline packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:151]: does this have a default value?

Comment (adamweeks) [reply packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:151]: Yes, 25.

Comment (adamweeks) [reply packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:151]: https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/widget-recents/src/container.js#L87

**Review by taymoork2 — APPROVED** [review]
LGTM

File: packages/node_modules/@webex/widget-recents/src/enhancers/setup.js [hunk 2]
```diff
@@ -128,12 +130,14 @@ function getInitialSpaces(props) {
     widgetStatus
   } = props;
 
+  const spaceLoadCount = props.spaceLoadCount || DEFAULT_SPACE_COUNT;
```
Comment (mwegman) [inline packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:133]:
```suggestion
  const {spaceLoadCount = DEFAULT_SPACE_COUNT} = props;
```

**Review by mwegman — APPROVED** [review]
Very minor comment

File: packages/node_modules/@webex/widget-recents/src/enhancers/setup.js (hunk 2 above)
Comment (taymoork2) [reply packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:133]: I was going to suggest this, but I couldn't remember if this handles updates/new props?

Comment (mwegman) [inline packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:133]:
```suggestion
  const spaceLoadCount = (props.spaceLoadCount > 0) ? props.spaceLoadCount : DEFAULT_SPACE_COUNT;
```
if the user passes in `'not the correct value'` or any other string

Comment (taymoork2) [reply packages/node_modules/@webex/widget-recents/src/enhancers/setup.js:133]: At what point does this become over-engineering? lol

**Pull request merged**